src/catkin_pkg/package_version.py

```diff
@@ -140,7 +174,7 @@ def get_forthcoming_label(rst):
             if len(section.children) > 0 and isinstance(section.children[0], docutils.nodes.title):
                 title = section.children[0]
         if title and len(title.children) > 0 and isinstance(title.children[0], docutils.nodes.Text):
-            title_text = title.children[0].rawsource
+            title_text = title.children[0].astext()
```

Comment:
Could you explain this change and its motivation?

Reply:
Race condition in comments. It's explained in another thread.

Reply:
Sorry, was commenting on this one as you were reviewing: https://github.com/ros-infrastructure/catkin_pkg/pull/331#discussion_r848937009